azkaban-db/src/main/sql/upgrade.3.85.0.to.3.86.0.sql

```diff
@@ -1,3 +1,4 @@
 -- DB Migration from release 3.85.0 to 3.86.0
 -- Adding ExecutionSource column in execution_flows
 alter table execution_flows add column execution_source varchar(32) default null;
+ALTER TABLE execution_flows ADD COLUMN execution_source VARCHAR(32) DEFAULT NULL;
```

Comment:
This line was most likely supposed to replace the existing one? Functionally it's exactly the same, just uppercase keywords.

Reply:
yes, thanks!